libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.LogAction.schema

```diff
@@ -1,8 +1,8 @@
 {
     "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/4.Future/schemas/component.schema",
     "$role": "unionType(Microsoft.IDialog)",
-    "title": "Log Action",
-    "description": "This is a action which writes to console.log and optional creates a TraceActivity around a text binding",
+    "title": "Log to console",
+    "description": "Log a message to the console and send a TraceActivity (optional).",
```

Comment:
>console [](start = 41, length = 7)

should not say to console.  
"Log a message to host application and send a TraceActivity to emulator.

Reply:
Updated. Dropped Emulator. 

"Log a message to the host application. Send a TraceActivity (optional)."